tools/tests/README.md

```diff
@@ -19,11 +19,42 @@ run the tests in Google Chrome.
 
 # Adding new tests
 
-All tests are located in tools/tests. There is a separate folder for each component.
-The name of the folder should be the same as the name of the folder that contains the
-source of the component. For example the test folder for the radial-menu should be radial-menu.
-This is required in order to simplify the setup process that copies the source bundles into
-the test folders. All tests are named **test.js**.
+All tests are located in tools/tests. There is a separate folder for each component. All tests are named **test.js**.

```

Comment:
All the tests are not named test.js, aren't they? There are components with multiple test files

Reply:
Yes, they are not. Changed.